### PR TITLE
AWS Bill PDF to CSV sample code

### PR DESCRIPTION
**Pull request by cjkindel**

### CODE REVIEW

File: aws-bill-pdf-to-csv/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+This sample takes in a S3 URI pointing at a AWS Bill PDF and converts the bill into a CSV. Each item in the bill is converted to a row in the CSV. It will then write the CSV back to a provided S3 URI. This showcases how Griptape can be used to automate previously manual processes such as identifying if a PDF field with identical formatting is a `AWS Region` or an `AWS Service`.
+
+[![Deploy_to_Griptape](https://github.com/griptape-ai/griptape-cloud/assets/2302515/4fd57873-5c93-44a8-8fa3-ac1bf7d73bcc)](https://cloud.griptape.ai/structures/create?sample-name=griptape-csv-filter&type=sample)
```
Comment (emjay07) [inline aws-bill-pdf-to-csv/README.md:3]: update the sample-name to the name of the folder

File: aws-bill-pdf-to-csv/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+This sample takes in a S3 URI pointing at a AWS Bill PDF and converts the bill into a CSV. Each item in the bill is converted to a row in the CSV. It will then write the CSV back to a provided S3 URI. This showcases how Griptape can be used to automate previously manual processes such as identifying if a PDF field with identical formatting is a `AWS Region` or an `AWS Service`.
```
Comment (emjay07) [inline aws-bill-pdf-to-csv/README.md:1]: nit: may want to call out that somewhere in here or in the Requirements section that the AWS permissions provided should then have read/write access to both S3 URIs.

**Comment by emjay07** [comment]
Oh, I forgot a note. Add to the table in the base README as well.

File: aws-bill-pdf-to-csv/README.md (hunk 1 above)
Comment (cjkindel) [reply aws-bill-pdf-to-csv/README.md]: Added (and renamed)

File: aws-bill-pdf-to-csv/README.md (hunk 2 above)
Comment (cjkindel) [reply aws-bill-pdf-to-csv/README.md]: Added links in **Configuration** section

**Review by emjay07 — APPROVED** [review]

**Pull request merged**